lnrpc/rpc.proto

```diff
@@ -159,6 +170,21 @@ message UnlockWalletRequest {
 }
 message UnlockWalletResponse {}
 
+message ChangePasswordRequest {
+    /**
+    current_password should be the current valid passphrase used to unlock the
```

Comment:
nit: the field name is not usually named like this in the proto comments, as it will not be consistent when the names are translated for `go`. No blocker though.

Reply:
Ah, didn't know. I was just going by the style used throughout the other messages.